### PR TITLE
Correct typo: Developping vs developing

### PR DESCRIPTION
**Pull request by CloCkWeRX**
**Checklist:**

Make sure you've done all the following (You can delete the checklist before submitting)

- [ ] If you have multiple commits please combine them into one commit by squashing them.

Tricky to do via the github edit UI - squash merge?

- [X] Read and understood the [contribution guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/CONTRIBUTING.md)

**Description:**

Typo in the translations. Not sure if this should be via crowdin, but when I tried through there, couldn't see the strings to fix.

### CODE REVIEW

File: po/common/en_AU.po [hunk 1]
```diff
@@ -4462,11 +4462,11 @@ msgid "Not a fair trade product"
 msgstr ""
 
 msgctxt "attribute_labels_fair_trade_yes_description_short"
-msgid "Helps producers in developping countries."
+msgid "Helps producers in developing countries."
```
Comment (CloCkWeRX) [inline po/common/en_AU.po:4465]: Github's code search didn't reveal other occurences of this id, but there may be more that need tweaking.

**Comment by CloCkWeRX** [comment]
https://github.com/openfoodfacts/openfoodfacts-server/pull/6294 might be the better place to deal with this.

**Comment by stephanegigandet** [comment]
Thanks for the correction @CloCkWeRX ! The files in the /po/common/ directory are generated by Crowdin from the common.pot file, so we just need to correct the common.pot files, and the other files will be corrected automatically.

**Review by teolemon — APPROVED** [review]

**Comment by teolemon** [comment]
Those are locales not updated automatically. We might delete them or tweak the Crowdin config one day.

**Comment by alexgarel** [comment]
Thanks @CloCkWeRX !

I've done a new PR replacing thanks to a thorough search: https://github.com/openfoodfacts/openfoodfacts-server/pull/6324

**Pull request closed without merging**